terraformize/terraformize_terraform_wrapper.py

```diff
@@ -40,25 +46,19 @@ def apply(self, variables: Optional[dict] = None, parallelism: int = 10) -> Tupl
             :return stdout: the stdout stream of the terraform apply
             :return stderr: the stderr stream of the terraform apply
         """
-        if variables is None:
-            variables = {}
-
         return_code, stdout, stderr = self.tf.apply(no_color=IsFlagged, var=variables, skip_plan=True,
                                                     parallelism=parallelism)
         return return_code, stdout, stderr
 
-    def destroy(self, variables: Optional[dict] = None, parallelism: int = 10) -> Tuple[str, str, str]:
+    def destroy(self, variables: Optional[dict] = {}, parallelism: int = 10) -> Tuple[str, str, str]:
```

Comment:
passing an empty dict `{}` as the default value in a function in python is considered a bad idea do to how python saves data in memory.

please set it to None & have 
```
        if variables is None:
            variables = {}
```
if it really needs to return an empty dict